.github/workflows/validate-nginx-config.yml

```diff
@@ -0,0 +1,74 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      custom_template_values:
+        default: '{}'
```

Comment:
I really don't like this, but GH actions doesn't provide something like map input type or something...

Reply:
@joshuagibeonm unfortunately the only supported type are number, string, and boolean.

See: https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype